docs/product/dev-toolbar/setup.mdx

```diff
@@ -36,6 +36,10 @@ You need to complete two steps to get the toolbar rendered on the page:
     </html>
     ```
 
+3. Edit [Project Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/projects/) in Sentry so the Toolbar is allowed to connect from your dev, staging, and production domains.
```

Comment:
This just redirects me to a random org (I thought it used to ask you to select an org?)

Reply:
it should take you to the last org you used, something like that